test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/SendingRequestEventVerifier.cs

```diff
@@ -241,18 +242,18 @@ internal void VerifySendingRequest(object sender, SendingRequest2EventArgs args)
                         this.assert.AreSame(this.expectedContext.Credentials, httpWebRequest.Credentials, "Request credentials were not set");
                     }
 #else
-                        this.assert.AreSame(this.expectedContext.Credentials, httpWebRequest.Credentials, "Request credentials were not set");
+                        this.assert.AreSame(this.expectedContext.Credentials, requestMessage.Credentials, "Request credentials were not set");
 #endif
 
                         this.assert.IsNull(this.uriFromEvent, "Last uri unexpectedly not null. Test hook did not fire");
                         this.assert.AreEqual(requestMessage.Url, httpWebRequest.RequestUri, "Request Uri does not match RequestUri from HttpWebRequest");
                         this.uriFromEvent = httpWebRequest.RequestUri;
 
                         this.assert.IsNull(this.methodFromEvent, "Last method unexpectedly not null. Test hook did not fire");
-                        this.methodFromEvent = httpWebRequest.Method;
+                        this.methodFromEvent = httpWebRequest.Method.ToString();
 
                         this.assert.IsNull(this.originalHeadersFromEvent, "Last headers unexpectedly not null. Test hook did not fire");
-                        this.assert.AreEqual(requestMessage.Headers.Count(), httpWebRequest.Headers.Count, "Request Headers count does not match Headers count from HttpWebRequest");
+                        this.assert.AreEqual(requestMessage.Headers.Count(), httpWebRequest.Headers, "Request Headers count does not match Headers count from HttpWebRequest");

```

Comment:
We seem to refer to HttpWebRequest quite a bit in this file yet we're switching away from that